### DISCUSSION TITLE
Contour Invariant VAE

### DISCUSSION THREAD

**Discussion by jelrod14** [DrCoffey/DeepSqueak#203] [Q&A] [answered]
Hi, 
I was wondering if there are any paper references or more information that I could read up on/ one of the deepsqueak team members could tell me about the contour invariant VAE clustering method. So far my literature searches haven't yielded much, and I'd like to understand the architecture, reasoning, etc for this option, as well as how its different/better than the kmeans that was previously recommended . Thanks in advance!
Jacob

**Comment by NickleDave** [comment, marked as answer]
You might find it useful to compare with
- https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1008228
- https://github.com/timsainb/AVGN
- https://elifesciences.org/articles/67855
- https://github.com/pearsonlab/autoencoded-vocal-analysis

if you're not already
